test/Quiz.test.js

```diff
@@ -0,0 +1,9 @@
+import test from 'ava'
+import React from 'react'
+import { shallow } from 'enzyme'
+import Quiz from '../src/Quiz'
+
+test('renders Hello', t => {
+  const w = shallow(<Quiz />)
```

Comment:
Good question! It's a way of testing React components in isolation, without rendering the components it contains. So React renders the component "one level deep". We can see what components it contains and what `props` they have, but they're not rendered (unless we explicitly tell it to render one of them).

Do you remember that React has a Virtual DOM? The other feature of shallow rendering is it only renders to the Virtual DOM, not the real DOM. This means we can 1. Access more information about the component, 2. Tests run faster since we don't need to run the them in a web browser.